figures/kCSD_properties/kCSD1D_test_fig1_and_fig2_v2.py

```diff
@@ -333,12 +333,13 @@ def main_loop(src_width, total_ele, inpos, lpos, nm, noise=0, srcs=1):
     total_ele = 32
     names = ['lc', 'cv']
     src_width = 0.001
-    noise_lvl = np.linspace(0, 0.5, 10)
+    noises = 3
```

Comment:
I like this code parametrization, but the name is not the most intuitive. How about:
```suggestion
    noise_lvl_n = 3
```
?